src/extension/tab/helpers.ts

```diff
@@ -64,7 +72,6 @@ export function getMutations(mutationsObj): QueryInfo[] {
     return {
       document: mutation,
       variables,
-      source: mutation?.loc?.source,
```

Comment:
Previously this `source` was used to print the `mutationString`, but was inconsistent with the way queries were printed, which just used the `print` function from `graphql` on the `document`. For consistency, this was updated to do the same.